exercises/pangram/lib/example.dart

```diff
@@ -0,0 +1,9 @@
+class Pangram {
+  bool isPangram(String sentence) {
```

Comment:
is Dart, is it better to have the name like `isPangram` or `is_pangram`?

Reply:
`isPanagram` is the preferred spelling :)

https://www.dartlang.org/guides/language/effective-dart/style#do-name-other-identifiers-using-lowercamelcase